SalesforceCommerceCloud/pwa-kit
# @W-16649600 E2E browser tests for DNT UI

These are the E2E browser tests for the DNT UI in PWA Kit. 

# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- (change1)

# How to Test-Drive This PR

- (step1) Pull this branch locally
- (step2) `export RETAIL_APP_HOME=http://localhost:3000`
- (step3) `npm run test:e2e:ui`
- (step4) See all tests pass


# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
